blank-canvas-blocks/assets/navigation.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * File navigation.js.
+ *
+ * Required to open the mobile navigation.
+ */
+(function () {
+
+	function addMobileMenuButton( navMenu ) {
+
+		const menuContainer = navMenu.querySelector( '.wp-block-navigation__container' );
+		const openButton = document.createElement( 'a' );
```

Comment:
Semantically, should this be a button instead of an `a`?

Reply:
Yes, absolutely.  Thank you.  I used to be quite the hound about that very thing... :P

Reply:
Refactor to button complete.